src/tests/utils/setup.ts

```diff
@@ -44,6 +52,65 @@ declare global {
      * @param sql - string containing sql to executein test DB
      */
     query: (sql: string) => Promise<unknown>;
+
+    /**
+     * Execute sql query, that creates new note in database with specified data
```

Comment:
Please make descriptions easier to read:


```suggestion
     * Inserts Note mock to the db
```

Reply:
changed